src/HomeKit/HMEnums.cs

```diff
@@ -1141,7 +1141,7 @@ public enum HMSignificantEvent {
 	}
 
 	[iOS (13,0), TV (13,0), Watch (6,0), NoMac]
-	public enum HMHomeManagerAuthorizationStatus : long {
+	public enum HMHomeManagerAuthorizationStatus : int {
```

Comment:
minor (no need to change it if it's complete) but
```csharp
public enum HMHomeManagerAuthorizationStatus {
```
is identical (since `int` is the default)

Reply:
Thanks